vite.config.mts

```diff
@@ -187,6 +187,8 @@ export default defineConfig(({ mode }) => {
               if (id.includes('gridplus-sdk')) return 'gridplus-sdk'
               if (id.includes('tronweb')) return 'tronweb'
               if (id.includes('viem')) return 'viem'
+              if (id.includes('@cetusprotocol')) return '@cetusprotocol'
+              if (id.includes('@mysten')) return '@mysten'
```

Comment:
@0xApotheosis Brain stopping to brain for tonight but if we want to give a try to this, took a look at recent additions (re: yarn.lock diff) and looks like:
- valibot version change brought by mysten added 1.74Mb
- `@0noco/graphqllsp` brought by mysten added 1.46Mb
- `graphql` brought by mysten added something similar to the above

Few others but actually smol diff

Obviously once web is built those Mb numbers are very different and we're really looking at the size of the main .js file in the build, but gut feel is tackling either valibot alone and/or the gql tooling packages may be the goer!